pymc/logprob/abstract.py

```diff
@@ -236,13 +236,16 @@ class ValuedRV(Op):
     and breaking the dependency of `b` on `a`. The new nodes isolate the graphs between conditioning points.
     """
 
+    view_map = {0: [0]}
```

Comment:
Needs view_map so PyTensor knows not to do inplace operations on this. The other Op already had the view_map defined